lib/sdk/content/l10n-html.js

```diff
@@ -16,6 +16,31 @@ const assetsURI = require('../self').data.url();
 
 const hideSheetUri = "data:text/css,:root {visibility: hidden !important;}";
 
+function translateElementAttributes(element) {
+  // Translateable attributes
+  const attrList = ['title', 'accesskey', 'alt', 'label', 'placeholder'],
```

Comment:
It isn't particularly well spelled out but we generally separate const declarations.